rails/app/controllers/admin/settings_controller.rb

```diff
@@ -79,97 +74,53 @@ def show
   end
 
   # GET /admin/settings/new
-  # GET /admin/settings/new.xml
   def new
-    # PUNDIT_REVIEW_AUTHORIZE
-    # PUNDIT_CHECK_AUTHORIZE
-    # authorize Admin::Setting
     @admin_settings = Admin::Settings.new
-    @scope = nil
-
-
-    respond_to do |format|
-      format.html # new.html.erb
-      format.xml  { render :xml => @admin_settings }
-    end
+    # renders new.html.haml
   end
 
   # GET /admin/settings/1/edit
   def edit
-    # PUNDIT_REVIEW_AUTHORIZE
-    # PUNDIT_CHECK_AUTHORIZE (did not find instance)
-    # authorize @setting
     @admin_settings = Admin::Settings.find(params[:id])
-
+    # NP: 2020-08-19 TODO: This is not working nut it doesn't seem critical
```

Comment:
I think we should either get this working or just take it out.
I'd start with figuring out if the `home/_project_info` is used anywhere else besides here.